src/types/redis_bloom_chain.cc

```diff
@@ -85,32 +108,14 @@ void BloomChain::createBloomFilterInBatch(const Slice &ns_key, BloomChainMetadat
   batch->Put(metadata_cf_handle_, ns_key, bloom_chain_meta_bytes);
 }
 
-void BloomChain::bloomAdd(const Slice &item, std::string *bf_data) {
+void BloomChain::bloomAdd(const uint64_t &item_hash, std::string *bf_data) {
```

Comment:
uint64_t is a small POD type, I think pass-by value for these small object is better